core/pom.xml

```diff
@@ -222,6 +222,18 @@
             <version>1.1.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.littleshoot</groupId>
+            <artifactId>littleproxy</artifactId>
+            <version>1.0.0-beta7</version>
```

Comment:
Oh I see it's only for tests. But then I'm wondering that it might cause issues like https://github.com/elastic/elasticsearch/pull/12171 ?